app/models/concerns/rubygem_searchable.rb

```diff
@@ -25,7 +25,8 @@ module RubygemSearchable
           suggest: { type: "completion", contexts: { name: "yanked", type: "category" } },
           yanked: { type: "boolean" },
           downloads: { type: "long" },
-          updated: { type: "date" }
+          updated: { type: "date" },
+          trusted_publisher: { type: "boolean" }
```

Comment:
Does this mapping change need full re-indexing?

Reply:
Good question. I would not think so, since there is no change to the database source schema, but I am not sure. It is my first time committing to this repo and I do not have full context, so I will wait for others to weigh in.

Reply:
Any new mappings do require a reindex, which is easy to do but needs some coordination when deployed.